tools/clang/test/CodeGenSPIRV_Lit/vk.layout.shader-record-ext.std430.hlsl

```diff
@@ -44,7 +44,7 @@ struct S {
 [[vk::shader_record_ext]]
 ConstantBuffer<S> cbuf;
 
-// CHECK: OpDecorate %type_ShaderRecordBufferEXT_S Block
```

Comment:
Maybe I'm paranoid, but shouldn't we check those decoration do apply to the type used for the cbuf variable declaration?
Since we are in fcgl, a bug could make us generate this type correctly, but not use it for the cbuf variable.

Reply:
Sure, added.